### PR TITLE
Adding SLO spec link to SLO reference page

### LINKED ISSUE #1528: Link to SLO spec from SLO ref page
SLO reference page should include a link to the SLO spec: https://github.com/keptn/spec/blob/master/service_level_objective.md

### PR DESCRIPTION
**Pull request by Amishakumari544**
Signed-off-by: Amishakumari544 <amishakumari544@gmail.com>

This PR fixes #1528

### CODE REVIEW

File: content/docs/1.0.x/reference/files/slo/index.md [hunk 1]
```diff
@@ -223,3 +223,5 @@ The pass and warning criteria for the `total_score` use the logical operator ">=
 
 * [Quality Gates](../../../define/quality-gates/)
 
+* [SLO](https://github.com/keptn/spec/blob/master/service_level_objective.md)
```
Comment (StackScribe) [inline content/docs/1.0.x/reference/files/slo/index.md:226]:
```suggestion
* [SLO specification](https://github.com/keptn/spec/blob/master/service_level_objective.md)
```

**Review by StackScribe — COMMENTED** [review]
A nit-picking comment.

File: content/docs/1.0.x/reference/files/slo/index.md (hunk 1 above)
Comment (Amishakumari544) [reply content/docs/1.0.x/reference/files/slo/index.md]: done!

**Review by thisthat — APPROVED** [review]

**Comment by thisthat** [comment]
Thank you @Amishakumari544 :) 
From the preview, everything looks good! I am merging this PR :shipit:

**Pull request merged**